tensorflow/core/api_def/base_api/api_def_ParallelInterleaveDatasetV2.pbtxt

```diff
@@ -1,6 +1,47 @@
 op {
   graph_op_name: "ParallelInterleaveDatasetV2"
   visibility: HIDDEN
+  in_arg {
+    name: "input_dataset"
+    description: <<END
+Dataset that produces a stream of arguments for the function `f`.
+END
+  }
+  in_arg {
+    name: "other_arguments"
+    description: <<END
+Additional arguments to pass to `f` beyond those produced by `input_dataset`.
+Evaluated once when the dataset is instantiated.
+END
+  }
+  in_arg {
+    name: "cycle_length"
+    description: <<END
+Number of datasets (each created by applying `f` to the elements of
+`input_dataset`) among which the `ParallelInterleaveDatasetV2` will cycle in a
+round-robin fashion.
+END
+  }
+  in_arg {
+    name: "block_length"
+    description: <<END
+Number of elements at a time to produce from each interleaved invocation of a
+dataset returned by `f`.
+END
+  }
+  in_arg {
+    name: "num_parallel_calls"
+    description: <<END
+If this input is >=1, it bounds the number of datasets from which this instance
```

Comment:
API documentation should not talk about implementation. It should described the behavior. From that perspective, I would reword this to:

Determines the number of threads that should be used for fetching data from input datasets in parallel. The Python API `tf.data.experimental.AUTOTUNE` constant can be used to indicate that the level of parallelism should be autotuned.